example/eblob_backend.c

```diff
@@ -976,7 +988,9 @@ static void eblob_backend_cleanup(void *priv)
 {
 	struct eblob_backend_config *c = priv;
 
-	eblob_cleanup(c->eblob);
+	if (c->eblob) {
```

Comment:
I've moved checking `c->eblob` intp `dnet_blob_config_cleanup` and now it will free only `c->data.file` if blob wasn't initialized.